lib/project_and_version_value.rb

```diff
@@ -37,7 +38,7 @@ def project_costs(proj)
       .select('MAX(spent_on) AS spent_on, SUM(hours) AS sum_hours')
       .where('start_date IS NOT NULL AND due_date IS NOT NULL')
       .joins(:time_entries)
-      .group('spent_on').collect { |issue| [issue.spent_on, issue.sum_hours] }
+      .group(:spent_on).collect { |issue| [issue.spent_on, issue.sum_hours] }
```

Comment:
Prefer `map` over `collect`.<br>Place the . on the previous line, together with the method call receiver.